tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "src/value/index.ts"
       ],
       "@sinclair/typebox": [
-        "src/typebox.ts"
+        "src/index.ts"
```

Comment:
Let's keep this as `src/typebox.ts` and remove `src/index.ts`. Deno users currently import using this name as a path so will keep it as is for the time being (although it is a little non-standard)